### DISCUSSION TITLE
Similar functionality for XML?

### DISCUSSION THREAD

**Discussion by Evynglais** [wnameless/json-flattener#69] [General]
This is a great library -- very useful for working with JSON and needing to identify all the nodes quickly and uniquely.

I'm also trying to do the same with XML; wondered if anybody has come across something similar?

**Comment by wnameless** [comment]
Actually there is a JSON library from org.json, which exists a XML.toJSONObject function allows you to convert XML to JSON. Then you can use the json-flattener library to do whatever you want.
